UserApi/UserApi/Settings.cs

```diff
@@ -22,5 +22,7 @@ public class AdGroup
         public string ProfessionalUser { get; set; }
         public string External { get; set; }
         public string JudicialOfficeHolder { get; set; }
+        public string PanelMember { get; set; }
```

Comment:
@virensareen userapi should not check for panel member etc as these are hearing roles. All it need to do is return JOH role if the user has AD group JOH

Reply:
UserProfileHelper just need to return JOH if the user has the JOH AD group